introduction/getting-started.qmd

```diff
@@ -108,9 +105,6 @@ We use the `+` operator to build from elements of a design to a design. @def-ch4
 
 ::: {#def-ch4num1 .declaration} 
 
-Two-arm randomized experiment
-
-![Two-arm randomized experiment declaration](/figures/declaration-example){#fig-designannotated}
```

Comment:
This doesn't look right to me. 

The text like this:

::: {#def-ch4num1 .declaration} 

Indicates the start of a declaration, which is ended by the line

:::

So you moved the declaration name (Two-arm randomized experiment) and the graphic of it out of the declaration. Can you revise and move back into declaration. 

To explain also, the diagnosis environment starts with:

::: {#lem-ch4num1} 

and ends with

:::

Reply:
I think I now understand how the text should look like. Let me know whether I got it right this time.